### PR TITLE
SHOW PROC '/statistic' with non-existent db id now fail loudly

### PR DESCRIPTION
**Pull request by mchades**
## What type of PR is this：
- [x] enhancement

## Problem Summary(Required) ：

When execute a `SHOW PROC '/statistic/dbid'` statement with non-existent db id like:
```
SHOW PROC '/statistic/12345';
```
It should thraw an error but follow
```
mysql> SHOW PROC '/statistic/12345';
+------------------+---------------------+----------------+
| UnhealthyTablets | InconsistentTablets | CloningTablets |
+------------------+---------------------+----------------+
| []               | []                  | []             |
+------------------+---------------------+----------------+
```

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 0 / 2 (00.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/common/proc/StatisticProcDir.java|0|2|00.00%|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 2 / 2 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/common/proc/StatisticProcDir.java|2|2|100.00%|

**Review by imay — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/java/com/starrocks/common/proc/StatisticProcDir.java [hunk 1]
```diff
@@ -233,6 +233,10 @@ public ProcNodeInterface lookup(String dbIdStr) throws AnalysisException {
             throw new AnalysisException("Invalid db id format: " + dbIdStr);
         }
 
+        if (catalog.getDb(dbId) == null) {
+            throw new AnalysisException("Invalid db id : " + dbIdStr);
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/common/proc/StatisticProcDir.java:237]:
```suggestion
            throw new AnalysisException("Invalid db id: " + dbIdStr);
```

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 2 / 2 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/common/proc/StatisticProcDir.java|2|2|100.00%|

**Review by wyb — APPROVED** [review]

**Review by imay — APPROVED** [review]

**Pull request merged**